tests/Phinx/Db/Adapter/PostgresAdapterTest.php

```diff
@@ -833,6 +833,51 @@ public function testChangeColumn()
         $this->assertTrue($this->adapter->hasColumn('t', 'column2'));
     }
 
+    public function testChangeColumnFromTextToSmallInteger()
```

Comment:
You can condense each of these different tests into a singular one using a [Data Provider](https://phpunit.readthedocs.io/en/9.3/writing-tests-for-phpunit.html#data-providers). There are examples of usage in other places in this file.

The data provider for this can return something like:
```
return [
  ['smallinteger', 32767],
  ['integer', 2147483647],
  ['biginteger', 9223372036854775807],
];
```

and then within the test, on the insert line, do `(string)$testValue` to get that string representation of the value.